.cargo/config.toml

```diff
@@ -0,0 +1,12 @@
+[alias]
+xtask = "run --package xtask --"
+x     = "xtask"
+
+[env]
+CARGO_WORKSPACE_DIR = { value = "", relative = true }
+
+[target.arm-unknown-linux-gnueabi]
```

Comment:
Out of curiosity, did you try targeting ARMv7? #88

Reply:
I have not yet tried it, no. It seems like something that should happen in a separate PR.